backend/src/api/auth/auth.controller.js

```diff
@@ -59,3 +66,13 @@ exports.loginUser = async (req, res, next) => {
     token: token,
   });
 };
+
+exports.logoutUser = async (req, res, next) => {
+  const tokenUser = req.get('Authorization').replace('Bearer ', '');
+  const user = await UserModel.findById(req.user._id);
```

Comment:
better to simply use .updateOne with $pull operator. It would be just two-lines then -> update and send response

Reply:
Done